app/controllers/SaveForLater.scala

```diff
@@ -47,21 +48,51 @@ object SaveForLater extends FrontendController {
 
   def saveForLater = RefNumAction.async { implicit request =>
     repository.findById(SessionId(hc), request.refNum).flatMap {
-      case Some(doc) =>
-        s4l(hc)(doc, hc).flatMap { pw =>
-          val sum = SummaryBuilder.build(doc)
-          audit(sum, pw)
-          val expiryDate = LocalDate.now.plusDays(playconfig.S4L.expiryDateInDays)
-          val email = sum.customerDetails.flatMap(_.contactDetails.email)
-          EmailConnector.sendEmail(sum.referenceNumber, sum.addressVOABelievesIsCorrect.postcode, email, expiryDate) map { _ =>
-            Ok(views.html.savedForLater(sum, pw, expiryDate))
+      case Some(doc) => {
+        val sum = SummaryBuilder.build(doc)
+        val expiryDate = LocalDate.now.plusDays(playconfig.S4L.expiryDateInDays)
+          if (doc.saveForLaterPassword.isDefined) {
+            playconfig.SaveForLater(doc.saveForLaterPassword.get)(hc)(doc, hc).flatMap { pw =>
+              audit(sum, pw)
```

Comment:
Audit password?

Reply:
yeah, was in the code previously but it seems that is not used at all, will remove and see if it still works

Reply:
it works, removed